docs/develop/cron-jobs.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Cron Jobs
+---
+
+You can use a `CronJob` to run jobs on a time-based schedule. These automated jobs run similarly to [Cron](https://en.wikipedia.org/wiki/Cron) tasks.
+
+Cron jobs are useful for creating periodic and recurring tasks, like running a database query or sending emails. Cron jobs can also schedule individual tasks for a specific time, such as if you want to schedule a job for a low activity period.
+
+<Callout type="important">
+
+**Cron jobs** have limitations and idiosyncrasies. For example, in certain circumstances, a single cron job can create multiple jobs. Therefore, jobs should be **idempotent**.
+
+</Callout>
+
+## Defining a Cron job
+
+With Encore you define cron jobs directly in your code, using our infrastructure declaration syntax. We believe it’s forward-compatible and has the easiest path to discoverability.
+
+```go
+package hello // service name
+
+import (
+	"context"
+	"encore.dev/cron"
+)
+
+/*
+  In this example, we are defining a cron job.
+  The comment above the cron job definition will 
+  be used as a description.
+
+  You can also add code snippets in your description,
+  which will be picked up by our parser and displayed
+  in our web UI.
+
+    var cfg := cron.JobConfig{
+      Name:     "Cron Job Example",
+      Every:    5 * time.Minute,
+      Endpoint: Cron
+    }
+*/
+var _ = cron.NewJob("cronjobexample", cron.JobConfig{
+	Name:     "Cron Job Example",
+	Every:    5 * time.Minute,
+	Endpoint: Cron,
+})
+
+//encore:api public path=/cron
+func Cron(ctx context.Context) (*Response, error) {
+	msg := "Hello, Cron!"
+	return &Response{Message: msg}, nil
+}
+
+type Response struct {
+	Message string
+}
+```
+
+<Callout type="important">
+
+We only support using infrastructure creation at the package level, any other call location will result in a compilation error.
```

Comment:
```suggestion
We only support creating cron jobs at the package level, any other call location will result in a compilation error.
```

I think we maybe need a separate page to talk about our infrastructure creation syntax. This page is specifically about cronjobs and people might get confused